tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -2110,11 +2175,19 @@ class MklQuantizedConvOp
              std::max(std::abs(max_filter[i]), std::abs(min_filter[i])));
       }
       dnnl::primitive_attr reorder_attr;
+#ifndef ENABLE_ONEDNN_V3
       if (depth == 1) {
         reorder_attr.set_output_scales(0, scales);
       } else {
         reorder_attr.set_output_scales(2, scales);
       }
+#else
+      // TODO(intel-tf): Enable this for int8 when using oneDNN v3.x
+      assert(depth == 1);
```

Comment:
Same here. Could you please replace `assert` with `DCHECK` and add a TODO to replace `DCHECK` with returning an error status?

Reply:
Done.